src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double sideLength;
+
+    public Square(Color color, double sideLength) {
+        super(color, Math.pow(sideLength, 2));
+        this.sideLength = sideLength;
+    }
+
+    @Override
+    public void draw() {
+        System.out.printf("Figure: square, side: %f, radius: %f, color: %s\n",
+                area,
+                sideLength,
+                color
+        );
```

Comment:
There seems to be a mistake in the `printf` method. The placeholder %f is used for the 'radius' which does not exist for a square. It should be 'side' instead, and the 'area' should be calculated and printed as well.